File: Dataset/Leetcode/test/94/291.cpp
 class Solution {
public:
    vector<int> XXX(TreeNode* root) {
    vector<int>  result;
    TreeNode* p;
	p = root;
    stack<TreeNode*> s;
	while (p || !s.empty())
	{
		if (p) 
		{ 
			s.push(p); 
			p = p->left; 
		}
		else
		{
			TreeNode* ret = s.top();
            s.pop();
            result.push_back(ret->val);
            p = ret->right;
		}
	}
	return result;
    }
};

undefined
for (i = 0; i < document.getElementsByTagName("code").length; i++) {     console.log(document.getElementsByTagName("code")[i].innerText); }

File: Dataset/Leetcode/test/26/291.java
 class Solution {
    public int XXX(int[] nums) {
        int p = 1;
        for(int i=1;i<nums.length;i++){
            if(nums[i] != nums[i-1]){
                nums[p++]=nums[i];
            }
        }
        return p;
    }
}

